tests/chainer_tests/distributions_tests/test_one_hot_categorical.py

```diff
@@ -0,0 +1,77 @@
+from chainer import cuda
+from chainer import distributions
+from chainer import testing
+import numpy
+
+
+@testing.parameterize(*testing.product({
+    'shape': [(2, 3), ()],
+    'is_variable': [True, False],
+    'sample_shape': [(3, 2), ()],
+    'k': [3],
+}))
+@testing.fix_random()
+@testing.with_requires('scipy')
```

Comment:
Would you apply the following change to use `scipy.stats.multinomial`?
```suggestion
@testing.with_requires('scipy>=0.19')
```